refarch-integrations/refarch-s3-integration/refarch-s3-integration-client-core/src/main/java/de/muenchen/refarch/integration/s3/client/domain/model/SupportedFileExtensions.java

```diff
@@ -10,6 +11,9 @@
  */
 @NoArgsConstructor
 public class SupportedFileExtensions extends HashMap<String, String> {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
Is serial ID required here? Are we (de)serializing these objects?

Reply:
Done

Reply:
No also removed it